lighthouse-core/report/html/templates.html

```diff
@@ -325,9 +331,72 @@
+  <div class="lh-lighthouse">
+    <svg width="717" height="189" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+      <defs>
+        <path id="a" d="M0 0h284v202H0z" />
+        <linearGradient x1="0%" y1="50%" x2="93.17%" y2="50%" id="c">
+          <stop stop-color="#F1F3F4" offset="0%" />
+          <stop stop-color="#FFF" offset="100%" />
+        </linearGradient>
+      </defs>
+      <g transform="translate(444 -13)" fill="none" fill-rule="evenodd">
+        <mask id="b" fill="#fff">
+          <use xlink:href="#a" />
+        </mask>
+        <g mask="url(#b)">
+          <g transform="translate(56 70)">
+            <path fill="#EC5548" d="M95 31h24v2H95z" />
+            <path fill="#FBC21B" d="M98 33h18v11H98z" />
+            <path fill="#EC5548" d="M95 43h24v7H95z" />
+            <path fill="#FFF" d="M97.63 50h19.74L120 97H95z" />
+            <path d="M107 22a10 10 0 0 1 10 10v1H97v-1a10 10 0 0 1 10-10zM96.77 66.23l20.97-10.7.63 11.87-22.28 11.87zM95 94.8L119.1 82l.9 14H95z"
+              fill="#EC5548" />
+            <path d="M0 132a177.07 177.07 0 0 1 108.5-37c40.78 0 78.38 13.78 108.5 37H0z" fill="#E8EAED" />
+            <rect fill="#FEF0C8" x="98" y="33" width="10" height="10" rx="5" />
+            <path fill="url(#c)" d="M7 0l91 33.18v10.05L7 77z" />
+          </g>
+        </g>
+        <g mask="url(#b)" fill="#FFF">
+          <g transform="translate(96 86)">
+            <circle cx="87.5" cy=".5" r="1" />
```

Comment:
these are the little stars right? should we just remove them? they look a little weird, deleting this whole block looks better IMO

| before | after |
| - | - |
| ![image](https://user-images.githubusercontent.com/2301202/40070368-b6810360-5822-11e8-9229-b12e91e46183.png) | ![image](https://user-images.githubusercontent.com/2301202/40070387-c5d56c52-5822-11e8-8052-4e7672bd8632.png)
 |